src/Todo/Model.hs

```diff
@@ -32,3 +42,58 @@ data Todo = Todo
     , todoDeadline    :: Maybe Deadline
     , todoDescription :: Maybe String
     } deriving (Eq, Show)
+
+data TodoDB = TodoDB 
+    { tododbTodos :: Todos
+    , tododbTags  :: [Tag]
```

Comment:
You can rename fields here to just `dbTodos` and `dbTags`. Or at least `todoDbTodos` and `todoDbTags`.